upgrade-instructions.md

```diff
@@ -19,7 +19,22 @@ Change list:
  - [Drop Cassandra schema version prior version 8](#drop-cassandra-schema-version-prior-version-8)
  - [Adopt UnboundID as a LDAP library](#adopt-unboundid-as-a-ldap-library)
  - [Review the architecture of the RabbitMQ event bus](#review-the-architecture-of-the-rabbitmq-event-bus)
+ - [Cassandra Schema update to V11](#cassandra-schema-update-to-v11)
  
+### Cassandra Schema update to V11
+
+Date 23/06/2021
+
+JIRA: https://issues.apache.org/jira/browse/JAMES-3516
+
+Concerned product: Distributed James
+
+Add threadId column to messageIdTable and imapUidTable in order to get a message's threadId.
+
+In order to benefit from this work, you need to [upgrade to the latest schema version](https://github.com/apache/james-project/blob/master/src/site/markdown/server/manage-webadmin.md#upgrading-to-the-latest-version).
```

Comment:
Wrong. There's no upgrades to do here.

However we need to document how to create the threadId collumn on existing tables (ALTER TABLE ...)

Reply:
By using cqlsh: https://cassandra.apache.org/doc/latest/tools/cqlsh.html

Data manipulation with CQL: https://cassandra.apache.org/doc/latest/cql/dml.html# -> for finding your commands